src/overlays/actors/ovl_En_Guard_Nuts/z_en_guard_nuts.h

```diff
@@ -2,16 +2,38 @@
 #define Z_EN_GUARD_NUTS_H
 
 #include "global.h"
+#include "objects/object_dnk/object_dnk.h"
 
 struct EnGuardNuts;
 
 typedef void (*EnGuardNutsActionFunc)(struct EnGuardNuts*, GlobalContext*);
 
+typedef enum {
+    /* 0 */ WAIT_ANIM,
```

Comment:
I would probably rename WAIT_ANIM to better define what it was, I was sure it was the animation for when they were hiding under the ground until I opened z64util and saw its the head cocked sideways watching you animation.

Maybe `HEAD_TILT_ANIM`? You could instead leave it with its current name but add `// Head tilted above ground animation.`?